scripts/manage_lambda_iam_role.py

```diff
@@ -1,68 +1,122 @@
-#! /usr/bin/env python3
-
+import argparse
 import json
+import logging
 import sys
 
 import boto3
 
 from azul import config
 from azul.deployment import aws
+from azul.logging import configure_script_logging
+
+log = logging.getLogger(__name__)
+
+
+def main(argv):
+    configure_script_logging(log)
+    parser = argparse.ArgumentParser(description='Manage lambda IMA roles')
```

Comment:
```suggestion
    parser = argparse.ArgumentParser(description='Manage lambda IAM roles')
```

Here and everywhere.